### PR TITLE
Created `stdlib` folder - no functionality change.

### PR DESCRIPTION
**Pull request by yebai**
This PR creates a `stdlib` folder for storing `distributions.jl` and `RandomMeasures.jl`.

### CODE REVIEW

File: src/utilities/Utilities.jl [hunk 1]
```diff
@@ -12,16 +12,10 @@ export  vectorize,
         Sample,
         Chain,
         init,
-        vectorize,
-        logpdf_binomial_logit,
```
Comment (xukai92) [inline src/utilities/Utilities.jl]: Are these not exported any more?

Comment (yebai) [reply src/utilities/Utilities.jl]: It seems these are only for internal uses. Any reason for exporting them?

Comment (cpfiffer) [reply src/utilities/Utilities.jl]: No reason I can see.

**Comment by yebai** [comment]
Ready to merge from my side.

**Pull request merged**